JS/Game.js

```diff
@@ -1,14 +1,39 @@
 class Game {
   constructor() {
     this.round = 0,
-    this.players = {}, // {player1: 'whatever is put in imput', ...}
+    this.players = [], // {player1: 'whatever is put in imput', ...}
     this.round3 = false,
     this.winner = null
+    this.data;
   }
 
   init() {
     //starts game
-    //this.round++
+    domUpdates.displayNames()
+    let playerOneName = $('.js-player1-name-input').val()
+    let playerTwoName = $('.js-player2-name-input').val()
+    let playerThreeName = $('.js-player3-name-input').val()
+    let playerOne = new Player(playerOneName);
+    this.players.push(playerOne)
+    let playerTwo = new Player(playerTwoName);
+    this.players.push(playerTwo)
+    let playerThree = new Player(playerThreeName);
+    this.players.push(playerThree)
+    this.round++
+  }
+
+  parseData() {
+    let clues = (Object.values(data)[2])
+
+    this.data = (Object.keys(Object.values(data)[1])).map((category, index) => {
```

Comment:
This is a little hard to follow...there's a bit too much going on with `Object.keys(Object.values(data)[1]))`

Can we break these out into variables that have semantic names to help me follow it a bit better? What does `data[1]` represent? What does `Object.values()` for that piece of information represent? What does `Object.keys()` for that information give us? Category Names?